src/main/java/gui/EditRulePopup.java

```diff
@@ -337,70 +359,72 @@ private JPanel createControlPanel() {
 	}
 
 	/**
-	 * Initializes all the JPanels at once.
+	 * Initialises all the JPanels at once.

```

Comment:
you corrected from american english to british english? XD I mean, nothing against, it just seems like a very particular thing

Reply:
Eclipse did it, I just applied it 😬